substrate/frame/broker/src/types.rs

```diff
@@ -263,8 +267,9 @@ pub struct ConfigRecord<BlockNumber, RelayBlockNumber> {
 	pub leadin_length: BlockNumber,
 	/// The length in timeslices of Regions which are up for sale in forthcoming sales.
 	pub region_length: Timeslice,
-	/// The proportion of cores available for sale which should be sold in order for the price
-	/// to remain the same in the next sale.
+	/// The proportion of cores available for sale which should be sold.
+	///
+	/// If more cores are sold than this, then the price will not be adjusted downwards further.
 	pub ideal_bulk_proportion: Perbill,
```

Comment:
A bit confused but didn't all those changes essentially remove the need for `ideal_bulk_proportion`? We no longer care if we hit our bulk target but simply track the pricing of sold bulk.

Reply:
I was indeed about to almost remove it, but hesitated as it kind of still makes sense. If you set this, you can prevent price corrections downwards. In other words you can have a soft cap - if more cores are sold, it's fine, but the price won't be corrected for sales that happened after the ideal bulk sale count.